public/css/home.css

```diff
@@ -4,3 +4,18 @@
   padding: 36px;
   color: black;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+.navbar {
+  transition-timing-function: ease;
+  transition: 0.5s;
+}
```

Comment:
We can move this to `css/index.css` since it's common across pages. Also include them in every page's html